corehq/ex-submodules/phonelog/models.py

```diff
@@ -19,6 +19,9 @@ class DeviceReportEntry(models.Model):
 
     class Meta:
         unique_together = [('xform_id', 'i')]
+        index_together = [
+            ("date", "domain"),
```

Comment:
I think this is saying the same thing I was saying: http://stackoverflow.com/questions/21753699/does-the-order-of-index-together-matter-in-a-django-model#21754118